src/containers/Tenant/ObjectSummary/ObjectSummary.tsx

```diff
@@ -57,19 +62,27 @@ const initialTenantCommonInfoState = {
     collapsed: getInitialIsSummaryCollapsed(),
 };
 
-function prepareOlapTableSchema(tableSchema: any) {
-    const {Name, Schema = {}} = tableSchema;
-    const {Columns, KeyColumnNames} = Schema;
-    const KeyColumnIds = KeyColumnNames?.map((name: string) => {
-        const column = Columns?.find((el: any) => el.Name === name);
-        return column.Id;
-    });
+// As result of checks isTableType && isColumnEntityType the incoming type will be TColumnTableDescription
```

Comment:
It seems for me that this comment may be a little bit confusing. You just declare that you need a `TColumnTableDescription` argument for this function and that's it! It doesn't really matter how you check it.